### PR TITLE
Distortion Model Testing

### PR DESCRIPTION
**Pull request by acpaquette**
Adds tests for the dawn and kaguya distortion models.

### CODE REVIEW

File: tests/DistortionTests.cpp [hunk 1]
```diff
@@ -177,3 +178,92 @@ TEST(DawnFc, testRemove) {
...
+  EXPECT_NEAR(dy, 1, 1e-8);
+}
+
+TEST(KaguyaTc, testRemove) {
+  csm::ImageCoord imagePt(1.0, 1.0);
+
+  double ux, uy;
+  double desiredPrecision = 0.0000001;
+  std::vector<double> distortionCoeffs = {1, 0, 0, 0,
+                                          1, 0, 0, 0};
```
Comment (jessemapel) [inline tests/DistortionTests.cpp]: We're only checking that the first term is being handled correctly, we should test non-zero coefficients for the other terms too.

Comment (acpaquette) [reply tests/DistortionTests.cpp]: I'm not sure how necessary it is to actually test the individual coefficients like this. The two tests bellow are applying and removing distortion with actual coefficients and all zero coefficients. Both of those seem to be covering all of our bases

File: tests/DistortionTests.cpp [hunk 2]
```diff
@@ -177,3 +178,79 @@ TEST(DawnFc, testRemove) {
...
+                  DistortionType::DAWNFC, desiredPrecision);
+
+  removeDistortion(dx, dy, ux, uy, coeffs,
+                  DistortionType::DAWNFC, desiredPrecision);
+
+
+  EXPECT_NEAR(dx,10.0,1e-8);
+  EXPECT_NEAR(dy,10.0,1e-8);
+  EXPECT_NEAR(ux,10.0,1e-8);
+  EXPECT_NEAR(uy,10.0,1e-8);
```
Comment (Kelvinrr) [inline tests/DistortionTests.cpp]: Any reason you are not using `ASSERT_DOUBLE_EQ`

File: tests/DistortionTests.cpp [hunk 3]
```diff
@@ -160,20 +160,97 @@ TEST(DawnFc, testApply) {
...
+  applyDistortion(imagePt.samp, imagePt.line, dx, dy, coeffs,
+                  DistortionType::KAGUYATC, desiredPrecision);
+
+  removeDistortion(dx, dy, ux, uy, coeffs,
+                  DistortionType::KAGUYATC, desiredPrecision);
+
+  EXPECT_NEAR(dx, 0.999566, 1e-6);
+  EXPECT_NEAR(dy, 1.00137, 1e-5);
+  EXPECT_NEAR(ux, 1.0, 1e-8);
+  EXPECT_NEAR(uy, 1.0, 1e-8);
```
Comment (Kelvinrr) [inline tests/DistortionTests.cpp]: Why weren't some able to be changed?

**Pull request merged**